package.json

```diff
@@ -55,15 +55,17 @@
     "devDependencies": {
         "@tsconfig/node18-strictest-esm": "^1.0.1",
         "@types/node": "^18.7.6",
-        "@typescript-eslint/eslint-plugin": "^5.62.0",
-        "@typescript-eslint/parser": "^5.62.0",
+        "@typescript-eslint/eslint-plugin": "6.21.0",
+        "@typescript-eslint/parser": "6.21.0",
         "concurrently": "^8.0.1",
-        "eslint": "^8.56.0",
-        "eslint-config-prettier": "^8.9.0",
-        "eslint-plugin-prettier": "^4.2.1",
+        "eslint": "8.56.0",
+        "eslint-config-prettier": "9.1.0",
+        "eslint-plugin-prettier": "5.1.3",
+        "eslint-plugin-react": "7.33.2",
+        "eslint-plugin-react-hooks": "4.6.0",
         "husky": "^8.0.3",
         "onchange": "^7.1.0",
-        "prettier": "^2.7.1",
+        "prettier": "3.2.5",
```

Comment:
Note: I pin the packages because the build is not stable right now and it can create discrepancy and errors that are hard to understand. 
Unless it's a standalone library that you install on another repo, it's rather unstable to have non-pinned dependency (like the issue I had with eslint error when I joined)